src/robot/running/keywordrunner.py

```diff
@@ -113,7 +114,12 @@ def _get_and_report_failure(self):
         return failure
 
 
-class ForLoopRunner(object):
+class BasicForLoopRunner(object):
```

Comment:
I'd either rename this to `ForInRunner` or have separate abstract class `ForRunnerBase` that all concrete for loop runners extend.